src/Annotations/AppKernel.php

```diff
@@ -103,6 +104,12 @@ public function __construct(string $environment = 'dev')
                 'authority' => '',
                 'group' => '',
             ],
+            'html_purifier' => ($config['html_purifier'] ?? []) + [
+                'Cache.SerializerPath' => __DIR__.'/../../var/cache/html_purifier',
```

Comment:
We need to create this directory in all environments: `/srv/annotations/var/cache/html_purifier`.

Reply:
Is the `Directory /srv/annotations/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer not writable, please chmod to 777` error related to this?

Reply:
It's related to that. I will work on `MarkdownSanitizerTest` so that it doesn't attempt to write to cache.

Reply:
https://github.com/elifesciences/annotations/pull/54